### PR TITLE
CY-231: Hide antivirus from Overview page for which no data is present.

### PR DESCRIPTION
**Pull request by VatsalJagani**

### CODE REVIEW

**Review by babhalodia — APPROVED** [review]

File: cyences_app_for_splunk/default/data/ui/views/cs_overview.xml [hunk 1]
```diff
@@ -238,30 +238,40 @@
...
+    [| search ((`cs_wineventlog_system` SourceName="Microsoft-Windows-Service Control Manager" Message="The Windows Defender Antivirus Service service entered the *" Message=*stopped*) OR (`cs_windows_defender` EventCode IN (1150, 1151))) earliest=$timeRange.earliest$ latest=$timeRange.latest$ | fields EventCode, host | stats latest(EventCode) as EventCode by host | stats count(eval(EventCode!=1150 AND EventCode!=1151)) as Count | search [| search `cs_windows_defender` earliest=$timeRange.earliest$ latest=$timeRange.latest$ | head 1 | stats count | eval search=if(count&gt;0, "", "DONOTRETURNANYRESULTS") | fields search | format] | eval Name="Currently Disabled/Stopped Windows Defender Services"]
```
Comment (babhalodia) [inline cyences_app_for_splunk/default/data/ui/views/cs_overview.xml:268]: | search ((`cs_wineventlog_system` SourceName="Microsoft-Windows-Service Control Manager" Message="The Windows Defender Antivirus Service service entered the *" Message=*stopped*) OR (`cs_windows_defender` EventCode IN (1150, 1151))) earliest=$timeRange.earliest$ latest=$timeRange.latest$  [| search `cs_windows_defender` earliest=$timeRange.earliest$ latest=$timeRange.latest$ | head 1 | stats count | eval search=if(count&gt;0, "", "sourcetype=DONOTRETURNANYRESULTS") | fields search | return $search]| fields EventCode, host | stats latest(EventCode) as EventCode by host | stats count(eval(EventCode!=1150 AND EventCode!=1151)) as Count   | eval Name="Currently Disabled/Stopped Windows Defender Services"

Comment (babhalodia) [reply cyences_app_for_splunk/default/data/ui/views/cs_overview.xml:268]: Bro Let's use something like above. More optimised query.

Comment (VatsalJagani) [reply cyences_app_for_splunk/default/data/ui/views/cs_overview.xml]: Done.

Comment (babhalodia) [reply cyences_app_for_splunk/default/data/ui/views/cs_overview.xml]: Bro, can we do these changes everyplace where we are using DONOTRETURNANYRESULTS on this dashboard.

Comment (VatsalJagani) [reply cyences_app_for_splunk/default/data/ui/views/cs_overview.xml]: Ohh, I see. Done

**Review by babhalodia — APPROVED** [review]

**Pull request merged**